Rules/CTF/Scripts/CTF.as

```diff
@@ -711,3 +711,11 @@ void onBlobDie(CRules@ this, CBlob@ blob)
 		}
 	}
 }
+
+void onBlobCreated(CRules@ this, CBlob@ blob)
+{
+	if (blob.getName() == "mat_gold")
+	{
+		blob.RemoveScript("DecayQuantity");
```

Comment:
It should include the file extension to avoid the chance of the file matcher finding the wrong file.
```suggestion
		blob.RemoveScript("DecayQuantity.as");
```